app/controllers/publications/overview/_base.js

```diff
@@ -42,6 +46,6 @@ export default class PublicationsOverviewBaseController extends Controller {
 
   @action
   reload() {
-    this.send('reloadModel');
```

Comment:
this is too aggresive, it also reloads the publications navbar.
replacing with a `this.router.refresh('publications.overview._base');`
It looks similar to a local `router.refresh()`, but it seems to be reloading route `publication.overview`.
It makes sense in that `_base` is not a real route but all routes do extend it.
Since we did not declare this route in `router.js`, the closest real route from `_base` is the parent `publication.overview`.

Reply:
also using in place rather than an action